include/shmem.h.in.tpl

```diff
@@ -99,6 +101,7 @@ typedef struct {
 #define SHMEM_CTX_SERIALIZED 0x001001L
 #define SHMEM_CTX_PRIVATE 0x001002L
 #define SHMEM_CTX_NOSTORE 0x001003L
+#define SHMEM_CTX_INVALID (-1L)
```

Comment:
@yfguo Could you please define this one the following way?:

```c
#define SHMEM_CTX_INVALID (shmem_team_t) (-1L)
```
Additionally, it believe you should define it AFTER you declare `typedef void* shmem_ctx_t;`, that is, before (or aftger) you define SHMEM_CTX_DEFAULT.

Moreover, I would argue that the following definition would be far better:
```c
#define SHMEM_CTX_INVALID NULL
```
At this point, I believe you may have confused `SHMEM_CTX_INVALID` with a context option handle, and not with the invalid handle as it should be.